docs/content/concepts/configuration/config-schema.mdx

```diff
@@ -7,19 +7,21 @@ description: Job run configuration allows providing parameters to jobs at the ti
 
 Run configuration allows providing parameters to jobs at the time they're executed.
 
+It's often useful to provide user-chosen values to Dagster jobs or software-defined assets at runtime. For example, you might want to choose what dataset an op runs against, or provide a connection URL for a database resource. Dagster exposes this functionality through a configuration API.
+
+Various Dagster entities (ops, assets, resources) can be individually configured. When launching a job that executes (ops), materializes (assets), or instantiates (resources) a configurable entity, you can provide _run configuration_ for each entity. Within the function that defines the entity, you can access the passed-in configuration off of the `context`. Typically, the provided run configuration values correspond to a _configuration schema_ attached to the op/asset/resource definition. Dagster validates the run configuration against the schema and proceeds only if validation is successful.
```

Comment:
I guess this just moved around and isn't new, but something about "instantiates (resources)" doesn't seem quite right to me, I don't think these nouns are uniquely determined by these verbs (materializing is basically the same thing as instantiating no?)

I think its more that the assets are what are produced by the job, and the resources are the external services that need to get brought in to produce them